en/developer_guide/smartAdmin.rst

```diff
@@ -0,0 +1,75 @@
+SmartAdmin
+==========
+
+Our back office use SmartAdmin with only some of JavaScript components (JarvisWidget and SmartNotification).
```

Comment:
As smartadmin may be unknown to your readers, can you present it ? What is it ? What about the licence ?